app/Http/Controllers/Order/InvoiceController.php

```diff
@@ -209,14 +207,14 @@ public function getInvoices(Request $request)
     public function show(Request $request)
     {
         try {
-            $id = $request->input('invoiceid');
-            $invoice = $this->invoice->where('id', $id)->first();
-            $invoiceItems = $this->invoiceItem->where('invoice_id', $id)->get();
+            $invoice = $this->invoice->where('id', $request->input('invoiceid'))->first();
+            $invoiceItems = $this->invoiceItem->where('invoice_id', $request->input('invoiceid'))->get();
```

Comment:
Please use select query if you need only few columns

Reply:
Fixed